test/runtests.jl

```diff
@@ -65,6 +65,13 @@ n, v = (40, 10), (4, 1)
         end
         @test Mr !== heppcat(Yb,k,T)
         @test Mr == heppcat(Yb,k,T)
+        
+        Mr = homppca(Yb,k)
+        for _ in 1:T
+            updateF!(Mr,Yb,ExpectationMaximization())
+        end
+        @test Mr !== heppcat(Yb,k,T;vknown=true)
+        @test Mr == heppcat(Yb,k,T;vknown=true)
```

Comment:
should one of these be `false` ??

Reply:
No, the first is just checking that the objects are not identical (perhaps not necessary). The second checks that their values match.

The preceding two checks (for the default `vknown=false`) do the same.

Reply:
oh, doh!